packages/web/components/navbar/index.tsx

```diff
@@ -150,7 +152,7 @@ const NavBarButton: FunctionComponent<
 };
 
 const SettingsDropdown: FunctionComponent<{
-  userSettings: IUserSetting[];
+  userSettings: UserSetting[];
```

Comment:
Hmm I think this is creating an empty div for the avatar user setting. This is an additional comment to the avatar user setting comment.

Reply:
Can you explain this a little bit more? I did not understand

Reply:
In that component, we map over `userSettings`, with the avatar setting containing no component or label, causing a div with no visible content to be created.

Reply:
Oh, got it. Will take a look!